deeplearning4j-core/src/test/java/org/deeplearning4j/gradientcheck/CNNGradientCheckTest.java

```diff
@@ -719,6 +754,74 @@ public void testSeparableConv2D() {
                 }
             }
         }
+    }
+
+    @Test
+    public void testSeparableConv2D() {
+        int nOut = 2;
+
+        int minibatchSize = 3;
```

Comment:
As above, test 1 & 3 (I realize this was already the case before this PR, but might as well change it here)